engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryKafkaAvroProvider.scala

```diff
@@ -12,12 +13,11 @@ class SchemaRegistryKafkaAvroProvider[T](schemaRegistryProvider: SchemaRegistryP
                                          topic: String,
                                          version: Option[Int]) extends KafkaAvroSchemaProvider[T] {
 
-  override def typeDefinition: typing.TypingResult =
-    AvroSchemaTypeDefinitionExtractor.typeDefinition(
-      schemaRegistryProvider.createSchemaRegistryClient.getSchema(
-        AvroUtils.valueSubject(topic), version
-      )
-    )
+  override def typeDefinition: Validated[SchemaRegistryError, typing.TypingResult] =
```

Comment:
Please add TODO to this class that it looks for version only during compilation, but during runtime it just parse a record using schema of this record.

Reply:
Done.